src/RouteJs/RouteJs.cs

```diff
@@ -35,6 +42,21 @@ IEnumerable<IConstraintsProcessor> constraintsProcessors
 		}
 
 		/// <summary>
+		/// Gets or sets whether urls are converted to lower case.
+		/// </summary>
+		public static bool LowerCaseUrls { get; set; }
```

Comment:
I tried to do just that at first, but strangely I found no `LowerCaseUrl` prop on the `RouteCollection` here. Any thoughts?